src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/govukCookieBanner.scala.html

```diff
@@ -41,7 +41,19 @@ <h2 class="govuk-cookie-banner__heading govuk-heading-m">
       @message.actions.map { actions =>
         <div class="govuk-button-group">
           @for(action <- actions) {
-            @action.href.foldNonEmpty {
+            @if(action.href.exists(_.nonEmpty)) {
+               @if(action.inputType.exists(_ == "button")) {
+                   @govukButton(Button(
```

Comment:
Is there a way to do this logic without duplicating the govukButton code?

Reply:
Had a look at the govuk-frontend code, and seen that the govukButton is duplicated there so probably best to go with that in the interests of parity with the Nunjucks code.